repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -91,7 +91,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
       inputStream.close()
       Some(defineClass(name, bytes, 0, bytes.length))
     } catch {
-      case e: Exception => None
+      case e: FileNotFoundException =>
+        // We did not find the class
+        logDebug(s"Did not load class $name from REPL class server at $uri", e)
```

Comment:
This shouldn't be debug level? Or is it a common case? What is the reference to a class server?